package.json

```
@@ -2,18 +2,20 @@
   "name": "robot3",
   "version": "0.2.0",
   "description": "A function, immutable Finite State Machine library",
-  "main": "machine.js",
+  "main": "dist/machine.js",
   "module": "machine.js",
   "type": "module",
   "files": [
+    "dist/",
     "debug.js",
     "machine.js"
   ],
   "scripts": {
     "minify": "terser machine.js -o machine.min.js",
     "bundlesize": "npm run minify && bundlesize --config bundlesize.json",
     "server": "http-server -p 1965",
-    "test": "node-qunit-puppeteer http://localhost:1965/test/test.html 10000"
+    "test": "node-qunit-puppeteer http://localhost:1965/test/test.html 10000",
+    "build": "rollup -d dist -f cjs machine.js debug.js"
```

Would you add this as well:

```suggestion
    "build": "rollup -d dist -f cjs machine.js debug.js",
    "preversion": "npm run build"
```